test/dtlstest.c

```diff
@@ -40,13 +41,27 @@ static unsigned char certstatus[] = {
 
 #define RECORD_SEQUENCE 10
 
+static unsigned timer_cb(SSL *s, unsigned timer_us)
+{
+    (void)s;
```

Comment:
This shouldn't be necessary. We have lots of functions with unused args where we don't do this, so I'm not sure why we should start now.